packages/camera/camera_platform_interface/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 2.4
```

Comment:
Some tests are failing because you are using `2.4` and not `2.4.0`.

Reply:
Yikes thanks I fixed